[2025-01-29T01:46:23.593+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T01:46:23.604+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_conditional_branching.determine_branch manual__2025-01-29T01:46:20.747753+00:00 [queued]>
[2025-01-29T01:46:23.609+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_conditional_branching.determine_branch manual__2025-01-29T01:46:20.747753+00:00 [queued]>
[2025-01-29T01:46:23.611+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-29T01:46:23.620+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): determine_branch> on 2025-01-29 01:46:20.747753+00:00
[2025-01-29T01:46:23.626+0000] {standard_task_runner.py:72} INFO - Started process 409 to run task
[2025-01-29T01:46:23.629+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'executing_conditional_branching', 'determine_branch', 'manual__2025-01-29T01:46:20.747753+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/executing_conditional_branching.py', '--cfg-path', '/tmp/tmp5rac6ler']
[2025-01-29T01:46:23.631+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask determine_branch
[2025-01-29T01:46:23.663+0000] {task_command.py:467} INFO - Running <TaskInstance: executing_conditional_branching.determine_branch manual__2025-01-29T01:46:20.747753+00:00 [running]> on host ef9d5dc750d2
[2025-01-29T01:46:23.854+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='executing_conditional_branching' AIRFLOW_CTX_TASK_ID='determine_branch' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T01:46:20.747753+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T01:46:20.747753+00:00'
[2025-01-29T01:46:23.855+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T01:46:23.862+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-01-29T01:46:23.869+0000] {python.py:240} INFO - Done. Returned value was: groupby_region_smoker
[2025-01-29T01:46:23.870+0000] {branch.py:38} INFO - Branch into groupby_region_smoker
[2025-01-29T01:46:23.870+0000] {skipmixin.py:230} INFO - Following branch ('groupby_region_smoker',)
[2025-01-29T01:46:23.880+0000] {skipmixin.py:278} INFO - Skipping tasks [('filter_by_southwest', -1), ('filter_by_northeast', -1), ('filter_by_northwest', -1), ('filter_by_southeast', -1)]
[2025-01-29T01:46:23.903+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T01:46:23.904+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=executing_conditional_branching, task_id=determine_branch, run_id=manual__2025-01-29T01:46:20.747753+00:00, execution_date=20250129T014620, start_date=20250129T014623, end_date=20250129T014623
[2025-01-29T01:46:23.923+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T01:46:23.940+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-29T01:46:23.944+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
